### ISSUE TITLE
Invalid js breaks testing

### ISSUE THREAD

**Issue by erichaus** [cogniteev/ember-cli-file-saver#1]
Hello,

The following line results in my test suite failing using this addon, is it possible to resolve?

https://github.com/cogniteev/ember-cli-file-saver/blob/2bd6d5024da9b0e82e78c686f66777465e5737ba/vendor/file-saver.shim.js#L6

```Built project successfully. Stored in "/code/tmp/core_object-tests_dist-W8WVblad.tmp".
not ok 1 PhantomJS 2.1 - Global error: SyntaxError: Unexpected token '}' at http://localhost:7357/assets/vendor.js, line 75518
    ---
        Log: |
            { type: 'error',
              text: 'SyntaxError: Unexpected token \'}\' at http://localhost:7357/assets/vendor.js, line 75518\n' }
    ...```

**Comment by panthony** [comment]
There it was ! Thanks for the report @erichonkanen I had trouble pin-pointing what was causing this issue on a project.

I released the version `0.0.7` with the fix.

Erratum: The version `0.0.8`.

**Comment by erichaus** [comment]
👏
